modules/vcf2maf/1.3/vcf2maf.smk

```diff
@@ -137,66 +161,162 @@ rule _vcf2maf_gnomad_filter_maf:
         touch {output.dropped_maf}
         """)
 
-def get_chain(wildcards):
-    if "38" in str({wildcards.genome_build}):
-        return reference_files("genomes/{genome_build}/chains/grch38/hg38ToHg19.over.chain")
+def get_original_genome(wildcards):
+    # Determine the original (i.e. input) reference genome for this sample
+    # Since this module projects to various output genome builds, we need to parse the sample table for the starting build
+    # To determine what we need to do
+    runs_table = config['lcr-modules']["vcf2maf"]["runs"]
```

Comment:
I just wonder if the sample table can be used here instead? Probably not so important, but using a sample table will allow to not use the normal id match and therefore decrease the number of checks. Pandas are very slow for large tables and it may take a long time for all of this to be computed if we try this even on the GAMBL-wide scale.

Reply:
I can take a look, or do some performance optimization